app/operations/assignments/create.rb

```diff
@@ -1,7 +1,7 @@
 module Assignments
   class Create < Operation
-    integer :base_project_id, default: Rails.application.secrets["zooniverse_project_id"]
```

Comment:
Pass in what now? The `project_id` and `workflow_id` ? WildCam Gorongosa Lab doesn't supply any project_ids at the moment, but if that's necessary to move forward, we will implement the change, no worries. Not sure what `workflow_id` would have to go in though, but I assume nothing would be fine?